### PR TITLE
SONAR-21453 Update the security contexts to use root as group ID

### PR DESCRIPTION
**Pull request by carminevassallo**

### CODE REVIEW

File: charts/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -40,6 +40,10 @@ annotations:
       description: "Run the initSysctl init-container as root to prevent 'permission denied' issues"
     - kind: changed
       description: "Add revisionHistoryLimit configuration for SonarQube application Deployment ReplicaSets & StatefulSets"
+    - kind: changed
+      description: "Update the security contexts to use root as group ID"
+    - kind: changed
+      description: "Change the compatible SonarQube version to 10.4.0"
```
Comment (jCOTINEAU) [inline charts/sonarqube/Chart.yaml:46]: i think this can be removed ?

File: charts/sonarqube-dce/Chart.yaml [hunk 2]
```diff
@@ -37,6 +37,10 @@ annotations:
       description: "Add revisionHistoryLimit configuration for SonarQube application Deployment ReplicaSets"
     - kind: deprecated
       description: "Introduce `ApplicationNodes.podDisruptionBudget` and `searchNodes.podDisruptionBudget` and deprecate `ApplicationNodes.podDistributionBudget` and `searchNodes.podDistributionBudget`."
+    - kind: changed
+      description: "Update the security contexts to use root as group ID"
+    - kind: changed
+      description: "Change the compatible SonarQube version to 10.4.0"
```
Comment (jCOTINEAU) [inline charts/sonarqube-dce/Chart.yaml:43]: i think this can be removed ?

**Review by jCOTINEAU — CHANGES_REQUESTED** [review]

File: charts/sonarqube/Chart.yaml (hunk 1 above)
Comment (carminevassallo) [reply charts/sonarqube/Chart.yaml]: yup, great catch!

**Review by jCOTINEAU — APPROVED** [review]
lgtm !

**Pull request merged**